package.json

```diff
@@ -37,6 +37,7 @@
   "dependencies": {
     "cheerio": "^1.0.0",
     "commander": "^12.1.0",
+    "entities": "^4.5.0",
```

Comment:
This is already a transitive dependency through cheerio. I chose this version because our version of cheerio depends on 4.5.0.